server.js

```diff
@@ -1,16 +0,0 @@
-const express = require("express");
-const app = express();
-const port = 3000;
-const studentRoutes = require("./src/students/routes");
-
```

Comment:
How to serve index.html files from here

Reply:
Next: How to start integrating React inside this project.

Reply:
> How to serve index.html files from here

app.use(express.static(path.join(__dirname, 'public')));

app.get('/', async(req, res) => {
    res.sendFile(path.join(__dirname, 'public', 'index.html'));
});